src/localization/messages_da.js

```diff
@@ -4,17 +4,31 @@
  */
 $.extend( $.validator.messages, {
 	required: "Dette felt er påkrævet.",
-	maxlength: $.validator.format( "Indtast højst {0} tegn." ),
-	minlength: $.validator.format( "Indtast mindst {0} tegn." ),
-	rangelength: $.validator.format( "Indtast mindst {0} og højst {1} tegn." ),
+	remote: "Ret venligst dette felt",
 	email: "Indtast en gyldig email-adresse.",
 	url: "Indtast en gyldig URL.",
 	date: "Indtast en gyldig dato.",
 	number: "Indtast et tal.",
 	digits: "Indtast kun cifre.",
+	creditcard: "Indtast et gyldigt kreditkortnummer.",
 	equalTo: "Indtast den samme værdi igen.",
+	time: "Angiv en gyldig tid mellem kl. 00:00 og 23:59.",
+	ipv4: "Angiv venligst en gyldig IPv4-adresse.",
+	ipv6: "Angiv venligst en gyldig IPv6-adresse.",
+	require_from_group:  $.validator.format("Angiv mindst {0} af disse felter."),
+	extension: "Indtast venligst en værdi med en gyldig endelse",
+	pattern: "Ugyldigt format",
+	lettersonly: "Angiv venligst kun bogstaver.",
+	nowhitespace: "Må ikke indholde mellemrum",
+	maxlength: $.validator.format( "Indtast højst {0} tegn." ),
+	minlength: $.validator.format( "Indtast mindst {0} tegn." ),
+	rangelength: $.validator.format( "Indtast mindst {0} og højst {1} tegn." ),
 	range: $.validator.format( "Angiv en værdi mellem {0} og {1}." ),
 	max: $.validator.format( "Angiv en værdi der højst er {0}." ),
 	min: $.validator.format( "Angiv en værdi der mindst er {0}." ),
-	creditcard: "Indtast et gyldigt kreditkortnummer."
+	minWords: $.validator.format("Indtast venligst mindst {0} ord"),
+	maxWords:  $.validator.format("Indtast venligst højst {0} ord"),
```

Comment:
Please add a space after `(` and before `)` in the above two lines.

Reply:
Oh sure, didn't see that error